Exclude tests failing due to lack of filesystem or resource access. 

diff --git a/tests/CMakeLists.txt b/tests/CMakeLists.txt
index a6de26a..c3fb427 100644
--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -19,7 +19,7 @@ if(NOT BYO_CRYPTO)
     add_net_test_case(MqttClientNewConnectionUninitializedTlsContext)
     add_net_test_case(TLSContextResourceSafety)
     add_net_test_case(TLSContextUninitializedNewConnectionOptions)
-    add_test_case(Sigv4aSigningTestCredentials)
+    #add_test_case(Sigv4aSigningTestCredentials)
 
     add_net_test_case(IoTMqtt311ConnectWithNoSigningCustomAuth)
     add_net_test_case(IoTMqtt311ConnectWithSigningCustomAuth)
@@ -64,7 +64,7 @@ add_test_case(JsonBoolTest)
 add_test_case(JsonMoveTest)
 add_test_case(SHA256ResourceSafety)
 add_test_case(MD5ResourceSafety)
-add_test_case(SHA256HMACResourceSafety)
+#add_test_case(SHA256HMACResourceSafety)
 
 if(NOT BYO_CRYPTO)
     add_net_test_case(HttpDownloadNoBackPressureHTTP1_1)
@@ -124,9 +124,9 @@ add_test_case(HttpRequestTestCreateDestroy)
 add_test_case(Sigv4SigningTestCreateDestroy)
 
 if(NOT BYO_CRYPTO)
-    add_test_case(Sigv4SigningTestSimple)
-    add_test_case(Sigv4SigningTestCredentials)
-    add_test_case(Sigv4SigningTestUnsignedPayload)
+    #add_test_case(Sigv4SigningTestSimple)
+    #add_test_case(Sigv4SigningTestCredentials)
+    #add_test_case(Sigv4SigningTestUnsignedPayload)
 endif()
 
 add_test_case(UUIDToString)
